eventually/authentication/models.py

```diff
@@ -102,32 +134,43 @@ def myattr_to_dict(self):
             'updated_at': int(self.updated_at.timestamp())
             }
 
-    #'FIXME: rewrite docs'
-    def update(self,
-               first_name=False,
-               last_name=False,
-               middle_name=False,
-               email=False,
-               password=False):
+    def update(self, first_name=None, last_name=None, middle_name=None, email=None, password=None):
         """
         Updates user profile in the database with the specified parameters.\n
         :param first_name: <str> User's first name
```

Comment:
indicate the type of the function parameter with sphinx.

Reply:
Fixed.